helpdesk_mgmt/models/helpdesk_ticket_team.py

```diff
@@ -18,7 +19,12 @@ class HelpdeskTeam(models.Model):
         default=lambda self: self.env['res.company']._company_default_get(
             'helpdesk.ticket')
     )
-
+    alias_id = fields.Many2one('mail.alias', string='Alias',
```

Comment:
mail.alias.mixin already defines this field:

https://github.com/odoo/odoo/blob/dd68d151426e971d393c173e43fd8a1f16ade294/addons/mail/models/mail_alias.py#L191

Why are you redefining it? If you want to add the help attribute, just put that one, not the rest. Also use a better layout for avoiding a lot of lines and no need to use `\`, but close on each line the quote and start a new one in the following, and they are concatenated (respect the space inside the quotes):
```
    alias_id = fields.Many2one(
         help="The email address associated with "
              "this channel...
```

Reply:
I think like you, but when checkin de code in the core, I can see this redefinition in all models... (hr_job, crm_team, project, maintenaince...)... anyway I prefer not to redefine attributes. Done :)

Reply:
@DarioLodeiros this is not a question of thinking one way or the other, but how ORM works. Odoo is not a model for checking for better practices, as it contains a lot of not updated code to latest features of the ORM. Full definition is going to work, but this forces you to:

- Retranslate terms.
- If upstream definition is updated through versions, you have to adapt the redefinition to that changes.

Reply:
@DarioLodeiros this is not a question of thinking one way or the other, but how ORM works. Odoo is not a model for checking for better practices, as it contains a lot of not updated code to latest features of the ORM. Full definition is going to work, but this forces you to:

* Retranslate terms.
* If upstream definition is updated through versions, you have to adapt the redefinition to that changes.

Reply:
Of course... I just wanted to say that it was guided by the references found ;)
Thanks Pedro!